### PR TITLE
add physics and shrinkWrap

### PR DESCRIPTION
**Pull request by yudaprama**
This is useful when rendering single/multiple Markdown inside ScrollView

### CODE REVIEW

**Comment by yudaprama** [comment]
I signed it!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.2.1
```
Comment (domesticmouse) [inline CHANGELOG.md]: Can you please take out the changelog entry? We will do that in a separate PR once we have all the changes together for a new release. Thanks!

**Comment by domesticmouse** [comment]
And rsolve conflicts, thanks!

**Comment by domesticmouse** [comment]
Closing this out as stale. Please feel free to re-open with the conflicts resolved. Thanks!
